storagemarket/provider_loop.go

```diff
@@ -98,7 +98,7 @@ func (p *Provider) processDealRequest(deal *types.ProviderDealState) (bool, stri
 		return false, "server error", fmt.Errorf("failed to insert deal in db: %w", err)
 	}
 
-	log.Infow("inserted deal into DB", "id", deal.DealUuid)
+	p.dealLogger.Infow(deal.DealUuid, "inserted deal into deals DB")
```

Comment:
Same here: do we need this log line?

Reply:
Yeah, this is nice to have when tracing the log to see what all worked before things failed.

Reply:
It's anyways just once per deal.